test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -980,11 +986,14 @@ public void SerializationTest(string filePath)
         /// This set of tests has slightly modified json where the id properties
         /// are altered when serialized to test deserialization of non-guid ids.
         /// </summary>
-        /// <param name="filePath">The path to a .dyn file. This parameter is supplied
-        /// by the test framework.</param>
+        /// <param name="fileId">A random guid assigned to a .dyn file. This parameter is supplied
+        /// by the test framework. You can get the file path by calling GetTestFileNameFromGuid(fileId).</param>
         [Test, TestCaseSource(nameof(FindWorkspaces)), Category("JsonTestExclude")]
-        public void SerializationNonGuidIdsTest(string filePath)
+        public void SerializationNonGuidIdsTest(string fileId)
```

Comment:
how do you know what this fileId (guid) is if it's being generated using `Guid.NewGuid()`?